Jetsnack/app/src/main/java/com/example/jetsnack/widget/CollectionsKtx.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetsnack.widget
+
+fun <K, V> MutableMap<K,V>.computeIfAbsent(key : K, calulation :(K)->V):V? {
```

Comment:
Is this needed? You could probably use `getOrPut`

https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.collections/get-or-put.html#

Reply:
This file wasn't used, deleted